integration-testsuite/common/src/test/java/org/jboss/wise/test/integration/incontainer/WiseIntegrationInContainerTest.java

```diff
@@ -45,26 +45,44 @@
 
     @Deployment(name = WAR, order = 1)
     public static WebArchive createDeploymentA() {
-        // retrieve a pre-built archive
-        WebArchive archive = ShrinkWrap
-                .create(ZipImporter.class, WAR + ".war")
-                .importFrom(new File(getTestResourcesDir() + "/../../../target/test-classes/" + WAR + ".war"))
-                .as(WebArchive.class);
-        return archive;
+       WebArchive archive = ShrinkWrap.create(WebArchive.class, WAR + ".war");
+       archive
+          .addClass(org.jboss.wise.test.integration.basic.HelloWorldInterface.class)
+          .addClass(org.jboss.wise.test.integration.basic.HelloWorldBean.class)
+          .setWebXML(new File(getTestResourcesDir() + "/WEB-INF/basic/web.xml"));
+       return archive;
     }
 
 
-    @Deployment(name = SERVLET_WAR, order = 2)
-    public static WebArchive createDeploymentB() {
-        // retrieve a pre-built archive
-        WebArchive archive = ShrinkWrap
-                .create(ZipImporter.class, SERVLET_WAR + ".war")
-                .importFrom(new File(getTestResourcesDir() + "/../../../target/test-classes/" + SERVLET_WAR + ".war"))
-                .as(WebArchive.class);
-        return archive;
-    }
+   @Deployment(name = SERVLET_WAR, order = 2)
+   public static WebArchive createDeploymentB() {
+      MavenResolverSystem resolver = Maven.resolver();
+
+      WebArchive archive = ShrinkWrap.create(WebArchive.class, SERVLET_WAR + ".war");
+      archive
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("commons-io:commons-io:1.4").withoutTransitivity().asSingleFile())
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("org.milyn:milyn-smooks-validation").withoutTransitivity().asSingleFile())
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("org.milyn:milyn-smooks-javabean").withoutTransitivity().asSingleFile())
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("org.milyn:milyn-smooks-rules").withoutTransitivity().asSingleFile())
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("org.milyn:milyn-smooks-core").withoutTransitivity().asSingleFile())
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("org.milyn:milyn-commons").withoutTransitivity().asSingleFile())
+         .addAsLibrary(resolver.loadPomFromFile(getTestResourcesDir() + "/../../../pom.xml").resolve("org.jboss.wise:wise-core-cxf:2.1.0-SNAPSHOT").withoutTransitivity().asSingleFile())
```

Comment:
This is possibly a problem, as it's going to break the test as soon as we move to a different wise version